channels/drive/client/drive_file.c

```diff
@@ -184,7 +184,7 @@ static BOOL drive_file_remove_dir(const WCHAR* path)
 static void drive_file_set_fullpath(DRIVE_FILE* file, const WCHAR* fullpath)
 {
 	free(file->fullpath);
-	file->fullpath = fullpath;
+	file->fullpath = (WCHAR*) fullpath;
 	file->filename = _wcsrchr(file->fullpath, L'/');
```

Comment:
@hardening you're right, taking the context into account it is better to drop the const modifier.